sensor/common/compliance/node_inventory_handler_impl.go

```diff
@@ -19,6 +24,15 @@ var (
 	errInventoryInputChanClosed = errors.New("channel receiving node inventories is closed")
 	errIndexInputChanClosed     = errors.New("channel receiving node indexes is closed")
 	errStartMoreThanOnce        = errors.New("unable to start the component more than once")
+	rhcosOSImageRegexp          = regexp.MustCompile(`(Red Hat Enterprise Linux) (CoreOS) ([0-9.-]+)`)
+	rhcosPackageId              = 666
```

Comment:
Could you explain to me the oddly specific number here? IIUC, it is just some starting point that we use as the package ID when the report is generated, right? 

If that is the case: As it's only ever used in one place, does it need to be exposed up here in the `var (` section?

Reply:
It is a magic number, yes. I chose it, cause there are many songs about it in the culture :D

I used it as a var to: (1) give it a name, so that it is less magical, and (2) increment it easily in case of collisions with existing IDs.
I am not sure what is a good starting value for that ID. I guess the lower number have higher chance of being taken, right? Maybe I could go with `2137`?

Reply:
I think the number is fine as is; maybe we can add a comment to explain what it's for? It primarily stood out to me, hence my question